homeassistant/components/switch/rest.py

```diff
@@ -69,8 +69,9 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the device on."""
+        headers = {'content-type': 'application/x-www-form-urlencoded'}
```

Comment:
Would it make more sense to allow this to be customizable via config? If not configured, it could always default to `Content-Type: application/x-www-form-urlencoded`

```yaml
switch:
  platform: rest
  resource: http://zoneminder_host/zm/api/monitors/1.json
  name: "Cam1"
  body_on: "Monitor[Function]=Modect&Monitor[Enabled]=1"
  body_off: "Monitor[Function]=Monitor&Monitor[Enabled]=1"
  headers:
    - name: Content-Type
      value: application/x-www-form-urlencoded
    - name: X-Authorization
      value: d41d8cd98f00b204e9800998ecf8427e
```